ruby-challenges.rb

```diff
@@ -32,3 +65,36 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+# PROCESS
+# use def for new method palin_string calling on (str)
+# make if statement for if the string equals string.reverse it's a palindrome, as it'd be the same forward or backward.
+# have it print using string interpolation that "#{} is a palindrome"
+# create elsif for the opposite
+# create else for catchall printing "error!"
+# code is not working correctly because the inputs have the first letter capitalized which would make Racecar not the same as racecaR
+# using downcase! we can permanantly make the strings all lowercased so we can properly see if they are the same forward and backward now
+# now our return is undercased so now we can use .capitalize in our string interpolation to return a properly capitalized output.
+
+def palin_string(str)
+    if str.downcase! == str.reverse
+        p "#{str.capitalize} is a palindrome"
```

Comment:
I really like your approach to downcase and then capitalizing in the string interpolation.